config-templates/ridge/ridge-csi-driver/1.0.0/deployment.yaml

```diff
@@ -0,0 +1,429 @@
+  annotations:
+    version: 1.0.0
+items:
+  - apiVersion: apps/v1
+    kind: StatefulSet
+    metadata:
+      name: csi-ridge-controller
+      namespace: kube-system
+      labels:
+        cloud.ridge.co/managed: 'true'
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: csi-ridge-controller
+      template:
+        metadata:
+          labels:
+            app: csi-ridge-controller
+            role: csi-ridge
+            cloud.ridge.co/managed: 'true'
+        spec:
+          volumes:
+            - name: socket-dir
+              emptyDir: { }
+            - name: vm-token
+              hostPath:
+                path: /opt/tectonic/etc/vm-token
+                type: File
+            - name: init-unit
+              hostPath:
+                path: /etc/systemd/system/app-init.service
+          containers:
+            - name: csi-provisioner
+              image: 'quay.io/k8scsi/csi-provisioner:v1.4.0'
+              args:
+                - '--csi-address=$(ADDRESS)'
+                - '--v=5'
+              env:
+                - name: ADDRESS
+                  value: /var/lib/csi/sockets/pluginproxy/csi.sock
+              volumeMounts:
+                - name: socket-dir
+                  mountPath: /var/lib/csi/sockets/pluginproxy/
+              imagePullPolicy: IfNotPresent
+            - name: csi-attacher
+              image: 'quay.io/k8scsi/csi-attacher:v3.1.0'
+              args:
+                - '--v=5'
+                - '--csi-address=$(ADDRESS)'
+              env:
+                - name: ADDRESS
+                  value: /var/lib/csi/sockets/pluginproxy/csi.sock
+              volumeMounts:
+                - name: socket-dir
+                  mountPath: /var/lib/csi/sockets/pluginproxy/
+              imagePullPolicy: IfNotPresent
+            - name: csi-ridge-plugin
+              image: eu.gcr.io/tectonic-devel/moonstone-csi-driver:{{ csi-version }}
+              args:
+                - /app/csi-driver
+                - --log-format=json
+                - --endpoint=$(CSI_ENDPOINT)
+                - --backend=https://tech.ridge.co/csi-backend/v1
+                - --driver-version={{ csi-version }}
+                - -v
+{{#offline-expansion}}
+                - '--offline-expansion'
+{{/offline-expansion}}
+{{#online-expansion}}
+                - '--online-expansion'
+{{/online-expansion}}
+              env:
+                - name: CSI_ENDPOINT
+                  value: 'unix:///var/lib/csi/sockets/pluginproxy/csi.sock'
+              volumeMounts:
+                - name: socket-dir
+                  mountPath: /var/lib/csi/sockets/pluginproxy/
+                - name: vm-token
+                  readOnly: true
+                  mountPath: /opt/tectonic/etc/vm-token
+                - name: init-unit
+                  readOnly: true
+                  mountPath: /etc/systemd/system/app-init.service
+              imagePullPolicy: Always
```

Comment:
is it required to have `imagePullPolicy: Always` or can it be changed to `imagePullPolicy: IfNotPresent`